src/dockerCompose/dockerComposeHoverProvider.ts

```diff
@@ -11,32 +11,25 @@ import parser = require('../parser');
 import suggestHelper = require('../utils/suggestSupportHelper');
 
 export class DockerComposeHoverProvider implements HoverProvider {
-    public _parser: parser.Parser;
-    public _keyInfo: KeyInfo;
-
     // Provide the parser you want to use as well as keyinfo dictionary.
-    public constructor(wordParser: parser.Parser, keyInfo: KeyInfo) {
-        this._parser = wordParser;
-        this._keyInfo = keyInfo;
+    public constructor(public readonly parser: parser.Parser, public readonly keyInfo: KeyInfo) {
```

Comment:
Slightly more interesting change; I don't want `public` variables to start with an underscore. That's weird.